modules/cmsis/cmsis_core_a_r_ext.h

```diff
@@ -29,6 +29,33 @@
 #define FSR_FS_ALIGNMENT_FAULT				(33)
 #define FSR_FS_DEBUG_EVENT				(34)
 #define FSR_FS_UNSUPPORTED_EXCLUSIVE_ACCESS_FAULT	(53)
+#elif defined(CONFIG_ARMV7_A)
+
+/**
+ * N.B.: these FSR encodings are only valid when the
+ * Short-descriptor translation table format is used
+ */
+
+#define FSR_FS_ALIGNMENT_FAULT					(1)
+#define FSR_FS_DEBUG_EVENT					(2)
+#define FSR_FS_ACCESS_FLAG_FAULT_1st_LEVEL			(3)
```

Comment:
Ditto: use all caps.
```suggestion
#define FSR_FS_ACCESS_FLAG_FAULT_1ST_LEVEL			(3)
```

Reply:
Done